repctl/pkg/cmd/failback.go

```diff
@@ -1,5 +1,5 @@
 /*
- Copyright © 2021-2022 Dell Inc. or its subsidiaries. All Rights Reserved.
+ Copyright © 2021-2023 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
Do we need to change copyright  here?

Reply:
I missed to update it in my previous PR, that's why :)